apis/v1alpha1/generator.yaml

```diff
@@ -1,3 +1,22 @@
-ignore:
-  resource_names:
-      - Secret
+resources:
+  Secret:
+    exceptions:
+      errors:
+        404:
+          # https://docs.aws.amazon.com/secretsmanager/latest/apireference/API_DescribeSecret.html#API_DescribeSecret_Errors
+          code: ResourceNotFoundException
+    is_arn_primary_key: true
+    renames:
+      # SecretId
+      #  The ARN or name of the secret.
+      #  For an ARN, we recommend that you specify a complete ARN rather than a partial ARN. See Finding a secret from a partial ARN.The ARN or name of the secret.
+      operations:
+        DescribeSecret:
+          input_fields:
+           SecretId: Name
```

Comment:
Would it be more appropriate to call this `ID`?

Reply:
address in [update generator.yaml:](https://github.com/aws-controllers-k8s/secretsmanager-controller/pull/2/commits/ded36c9c2f8262ee20f64147f5246ffaa59963ad)